.github/workflows/test_e2e.yml

```diff
@@ -32,6 +32,10 @@ jobs:
         run: npm install appium -g
       - name: Setup E2E Environment
         run: pip3 install -r requirements.txt
+        env:
+          HATS_DEVICE_FARM_REMOTE_URL: ${{ secrets.HATS_DEVICE_FARM_REMOTE_URL }}
+          HATS_DEVICE_FARM_USERNAME: ${{ secrets.HATS_DEVICE_FARM_USERNAME }}
+          HATS_DEVICE_FARM_API_KEY: ${{ secrets.HATS_DEVICE_FARM_API_KEY }}
```

Comment:
one question, @sebastianquek and @younglim, is this the right section to put it under ah? 

or should it be under the name name: `Run E2E Tests`, since this Setup just installs the requirements.txt and we wont reference to the variable yet.